website/config.toml

```diff
@@ -0,0 +1,162 @@
+baseURL = "/"
+title = "Cloud Operations Sandbox"
+
+staticDir = ["static"]
+
+enableRobotsTXT = true
+
+theme = ["docsy"]
```

Comment:
nit: maybe it's me, but I'm not a huge fan of the table of contents style of this theme. I find it hard to navigate

Reply:
Any suggestions on which theme would be more useful?

Reply:
Open to suggestions! The open source team recommended Docsy but since this is a Theme it should be easy to change. More than happy to roll out with this and change it in the future if we have a theme we prefer.

Reply:
Sorry I'm not familiar with the theme options. Docsy is probably fine since I've seen it in other Google products. I'm just not a fan of those nav bars